clang/tools/3c/utils/port_tools/compile_converted_project.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# usage: compile_converted_project.sh PATH_TO_clang [MAKE_ARGS...]
+# The working directory must be the project.
+
+clang="$1"
+shift
+
+find . -name '*.checked.*' | while read f_checked; do
+    [[ "$f_checked" =~ ^(.*)\.checked(\..*)$ ]] || {
+        echo >&2 "Weird filename: $f_checked"
+        continue
+    }
+    f="${BASH_REMATCH[1]}${BASH_REMATCH[2]}"
+    mv "$f_checked" "$f"
```

Comment:
IIUC, this is moving each `.checked.*` file over-top the original. Is that right? I don't know the `REMATCH` etc. syntax but the regexp match looks correct to me.

Reply:
Right, though this code is no longer used.